tests/integration/ambient/waypoint_test.go

```diff
@@ -605,3 +608,191 @@ spec:
 			)
 		})
 }
+
+func TestIngressToWaypoint(t *testing.T) {
```

Comment:
The feature flag defaults to on; can we test that ingress traffic does not traverse waypoints with the feature flag off? I know it'll require a `NewSuite` setup, but maybe we can combine it with some other edge cases?

Reply:
Maybe we do it in the unit test (which is a very high level unit test)?

Reply:
Better than nothing I supposed. I imagine the feedback loop would be very quick